CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+<a name="8.0.2"></a>
+## [8.0.2](https://github.com/NativeScript/nativescript-angular/compare/8.0.1...8.0.2) (2019-06-28)
+
+### Bug Fixes
+
+ * remove deprecated API usage ([#1874](https://github.com/NativeScript/nativescript-angular/pull/1874)) ([ab740cd](https://github.com/NativeScript/nativescript-angular/commit/ab740cd))
+ * migrate from ReflectiveInjector (deprecated) to StaticInjector ([#1868](https://github.com/NativeScript/nativescript-angular/pull/1868)) ([e432841](https://github.com/NativeScript/nativescript-angular/pull/1868/commits/e432841e9b474ad188f87044b74666b322d68b5d))
+
```

Comment:
we may include the update of the angular dependencies

Reply:
Done. Such changes are not registered by our automatic changelog tool. We need to resolve this in future releases. At the moment any commit if type "chore:" is ignored by the change log and we have always updated deps using "chore:". Maybe we need to start tagging this as "feature:"